babybear/src/main/java/frc/robot/subsystems/EveryBotPickerSubsystem.java

```diff
@@ -0,0 +1,145 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: EveryBotPickerSubsystem.java
+// Intent: Subsystem to model the every bot picker motor and associated subsystem.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.subsystems;
+
+import frc.robot.common.*;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.SparkMaxRelativeEncoder;
+import com.revrobotics.CANSparkMax.*;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+/**
+ * A class intended to model the picker - checked with Simeon on 02/15/2023 name is now 'picker'
+ */
+public class EveryBotPickerSubsystem extends SubsystemBase {
+
+    /* *********************************************************************
+    CONSTANTS
+    ************************************************************************/
+    // TODO - use something less than 1.0 for testing
+    private static final double neoMotorSpeedReductionFactor = 1.0;
+
+    // EveryBot picker gear reduction
+    // TODO - get proper values from Simeon/Grayson
+    private static final double everyBotGearReduction = 10.0/1.0;
+
+    /* *********************************************************************
+    MEMBERS
+    ************************************************************************/
+    private CANSparkMax everyBotMotor = new CANSparkMax(Constants.EveryBotPickerMotorCanId, MotorType.kBrushless);
```

Comment:
is this correct for the new fall 2023 robot?

Reply:
Already changed for wrist motor